src/test/java/org/broadinstitute/consent/http/service/DarCollectionServiceTest.java

```diff
@@ -306,17 +318,47 @@ void testCreateElectionsForDarCollection() throws Exception {
     dar.setReferenceId(UUID.randomUUID().toString());
     DarCollection collection = createMockCollections().get(0);
     collection.addDar(dar);
-    when(darCollectionServiceDAO.createElectionsForDarCollection(any(), any())).thenReturn(
+    when(darCollectionServiceDAO.createElectionsForDarByUser(any(), any())).thenReturn(
         List.of("electionId"));
     when(voteDAO.findVoteUsersByElectionReferenceIdList(any())).thenReturn(List.of(new User()));
 
     service.createElectionsForDarCollection(user, collection);
-    verify(darCollectionServiceDAO).createElectionsForDarCollection(any(), any());
+    verify(darCollectionServiceDAO, times(1)).createElectionsForDarByUser(any(), eq(dar));
+    verify(voteDAO, times(1)).findVoteUsersByElectionReferenceIdList(any());
+    verify(darCollectionDAO, times(1)).findDARCollectionByCollectionId(any());
+    verify(darCollectionServiceDAO).createElectionsForDarByUser(any(), any());
     verify(voteDAO).findVoteUsersByElectionReferenceIdList(any());
     verify(emailService).sendDarNewCollectionElectionMessage(any(), any());
     verify(darCollectionDAO).findDARCollectionByCollectionId(any());
   }
 
+  @Test
+  void testCreateElectionsForProgressReport() throws Exception {
+    User user = new User();
+    user.setEmail("email");
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setReferenceId(UUID.randomUUID().toString());
+    dar.setId(randomInt(1, 10));
+    dar.setSubmissionDate(Timestamp.from(Instant.now()));
+    DataAccessRequest progressReport = new DataAccessRequest();
+    progressReport.setReferenceId(UUID.randomUUID().toString());
+    progressReport.setParentId(dar.getId());
+    progressReport.setSubmissionDate(Timestamp.from(Instant.now()));
+    DarCollection collection = createMockCollections().get(0);
+    collection.addDar(dar);
+    collection.addDar(progressReport);
+
+    when(darCollectionServiceDAO.createElectionsForDarByUser(any(), any())).thenReturn(
+        List.of("electionId"));
+    when(voteDAO.findVoteUsersByElectionReferenceIdList(any())).thenReturn(List.of(new User()));
+
+    service.createElectionsForDarCollection(user, collection);
+    verify(darCollectionServiceDAO, times(1)).createElectionsForDarByUser(any(), eq(progressReport));
+    verify(voteDAO, times(1)).findVoteUsersByElectionReferenceIdList(any());
+    verify(darCollectionDAO, times(1)).findDARCollectionByCollectionId(any());
+    verify(emailService, times(1)).sendProgressReportNewCollectionElectionMessage(any(), any());
```

Comment:
This also has redundant `times(1)` calls. I know that our existing tests rely heavily on `any()`, but I think it's worth moving away from that for new tests
```suggestion
    User voteUser = new User();
    String electionId = "electionId";

    when(darCollectionServiceDAO.createElectionsForDarByUser(user, progressReport))
        .thenReturn(List.of(electionId));
    when(voteDAO.findVoteUsersByElectionReferenceIdList(List.of(electionId)))
        .thenReturn(List.of(voteUser));

    service.createElectionsForDarCollection(user, collection);
    verify(darCollectionDAO).findDARCollectionByCollectionId(collection.getDarCollectionId());
    verify(emailService)
        .sendProgressReportNewCollectionElectionMessage(List.of(voteUser), collection.getDarCode());
  }
```